Runners/ConsoleApp/ConsoleApp.csproj

```diff
@@ -5,6 +5,15 @@
     <TargetFramework>net6.0</TargetFramework>
   </PropertyGroup>
 
+  <ItemGroup>
+    <COMReference Include="{215d64d2-031c-33c7-96e3-61794cd1ee61}">
+      <WrapperTool>tlbimp</WrapperTool>
```

Comment:
Why reference tlbimp?

Reply:
Unnecessary.
Probably a result of the attempts to work with the resources